cypress/integration/admin/pageBuilder/page/createPage.spec.js

```diff
@@ -1,77 +1,63 @@
-//                 });
-//         });
-//
-//         cy.findByTestId("pb-page-details").within(() => {
-//             cy.findByTestId("pb-page-details-header-edit-revision").click();
-//         });
-//
-//         cy.findByTestId("pb-editor-back-button").click();
-//
-//         // Wait till the "/pages" route
-//         cy.findAllByTestId("new-record-button").first().should("exist");
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(newPageTitle).should("exist");
-//                     cy.findByText(/Static/i).should("exist");
-//                     cy.findByText(/Draft/i).should("exist");
-//                     cy.findByText(/\(v2\)/i).should("exist");
-//                 });
-//         });
-//
-//         // Delete page
-//         cy.findByTestId("pb-page-details-header-delete-button").click();
-//         cy.findByRole("alertdialog").within(() => {
-//             cy.findAllByTestId("dialog-accept").next().click();
-//         });
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(newPageTitle).should("not.exist");
-//                 });
-//         });
-//     });
-// });
+context("Page Builder Page Creation", () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    describe("When creating new page", () => {
+        const id = uniqid();
+
+        afterEach(() => {
+            cy.pbListPages({
+                search: {
+                    query: id
+                }
+            }).then(pages => {
+                pages.forEach(page => cy.pbDeletePage({ id: page.id }));
+            });
+        });
+
+        it("should be able to create page, publish page and check whether it exists in the list of the pages", () => {
+            const newPageTitle = `Test page ${id}`;
+
+            cy.visit("/page-builder/pages");
+
+            // Button is hidden so we need to force click it.
+            cy.findByTestId("new-page-button").click({ force: true });
+            // After clicking on create blank page button, we should be redirected to the page editor of that newly created page.
+            cy.findByTestId("create-blank-page-button").click();
+            // Check if we got redirected to the Page Builder Editor (route "/page-builder/editor/").
+            cy.url().should("includes", "/page-builder/editor/");
+            // Check if we are on the Page Builder Editor page (route "/page-builder/editor/") and it is empty.
+            cy.findByTestId("pb-content-add-block-button", { timeout: 15000 });
+
+            // Changing default name for the page (because default name is Undefined) so we can use it in the query to find it's id,
```

Comment:
```suggestion
            // Changing default name for the page (because default name is Untitled) so we can use it in the query to find its ID,
```